main/features.c

```diff
@@ -1011,7 +1011,7 @@ static int bridge_exec(struct ast_channel *chan, const char *data)
 			strlen(args.dest_chan));
 	}
 	if (!current_dest_chan) {
-		ast_verb(4, "Bridge failed because channel '%s' does not exist\n",
+		ast_log(LOG_WARNING, "Bridge failed because channel '%s' does not exist\n",
```

Comment:
This seems like an unrelated logging change

Reply:
It's related in that if bridging fails, it's now noticeable to the user. Before it was just a verbose, even though something went wrong.

But I'll put up another review at some point for log message related changes so if you'd rather it go with that, I can move it there.

Reply:
That's a separate thing and should be a separate review.